submissions/hello_world/DogOrCat/README.txt

```diff
@@ -0,0 +1,10 @@
+This program shows an image of a cat or a dog.
+
+*******
```

Comment:
You can use markdown section blocks here. See [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)